splunk_add_on_ucc_framework/ucc_ui_lib/src/main/webapp/components/TestComponent.jsx

```diff
@@ -21,12 +20,13 @@ class TestComponent extends Component {
     }
 
     componentDidMount() {
-        console.log('getUnifiedConfigs: ', getUnifiedConfigs());
         axiosCallWrapper(this.props.serviceName)
             .then((response) => {
+                // eslint-disable-next-line no-console
                 console.log(response.data);
```

Comment:
We don't need to log `globalConfig.json`. Please remove this. And in case of error please use `console.error`.